src/Mvc/Mvc.Core/src/ControllerBase.cs

```diff
@@ -217,6 +217,11 @@ public ProblemDetailsFactory ProblemDetailsFactory
     /// </summary>
     public ClaimsPrincipal User => HttpContext?.User!;
 
+    /// <summary>
+    /// Creates the <see cref="EmptyResult"/>
+    /// </summary>
+    public static EmptyResult Empty => new();
```

Comment:
According to the proposal in https://github.com/dotnet/aspnetcore/issues/40168#issuecomment-1054550507 it should be:
```suggestion
    public static EmptyResult Empty { get; } = new();
```

Reply:
isn't it the same thing!? Or am I missing something here!?

Reply:
`=>` is equivalent to `get { return ...; }` which means that it would return a new instances everytime. `{ get; } = ...;` assigns a single instance and thus returns the same instance everytime.

https://stackoverflow.com/a/38999936

Reply:
:( Seems like can't use it like this. Need to do like the old style. 
![image](https://user-images.githubusercontent.com/17148381/156539349-ab486847-597a-4bb3-88cb-05d11be30246.png)

Reply:
I got it. Thanks 
https://sharplab.io/#v2:D4AQTAjAsAUCDMACciDCB7AdgFwE7oBsCBTXAIQEMBnYxAb1kSeSRAgDZEBlbAVwBNiORADEAlrirZEAXgB8iTMQDuACgCUAbkbMEyDtz6DhXYgGMs/eogDmxbJsQBfWYpUbtMJ7B9xWYQwEhaQYYZkRYJyA